main/lsp/wrapper.cc

```diff
@@ -21,6 +29,11 @@ vector<unique_ptr<LSPMessage>> LSPWrapper::getLSPResponsesFor(unique_ptr<LSPMess
         initialized = true;
     }
 
+    // Retrieve any notifications that would normally be sent asynchronously in a multithreaded scenario.
```

Comment:
The output sent this way will be empty right now, but eventually will contain diagnostics when we move typechecking to a dedicated thread.